### ISSUE TITLE
Add Firebase as a deployment option to meet technical specifications of Project1.md

### ISSUE THREAD

**Issue by airbr** [ATL-WDI-Curriculum/projects#2]
# [Firebase Documentation on Deployment](https://firebase.google.com/docs/hosting/deploying)

The students have been introduced to Firebase by means of the recent Google Dev Fest Event held at General Assembly Atlanta Campus. I have personally used Firebase and cannot think of a particular technical reason why it is inappropriate whereas Github Pages or Bit-balloon is.

Firebase is an incredibly fast command line option for launching a project  assets such as a HTML/CSS/JS game to a live online URL. The only particular issue is the requirement being that have a google account (perhaps a cause for including, instructions how to set up a developer specific google account for this purpose).

With Firebase students can launch their Game Project with the most basic use of NodeJS. Students have already installed at this point in the course. There is an added step to deploy the website, but it involves an automatic process and just making sure that website assets are in public folder that need to be. Firebase also generates a .src file when it is initialized comparable to a git initialization instructions/history. 

Perhaps it could be presented a a slightly more advanced options if the Github Pages is deployed successfully, as both solutions directly run from the Git Repository just hosted with a different service.

I will be opening a Pull Request with markdown to this effect

**Comment by drmikeh** [comment]
Sounds fine to me.  Can you summarize the steps needed to deploy a static web site to Firebase (or include a link to such instructions)?

**Comment by drmikeh** [comment]
I accepted and merged the pull request adding this option.
